components/Footer/Footer.module.scss

```diff
@@ -282,8 +300,8 @@
     @include media-query(menu-break) {
         order: 2;
         margin-bottom: 0;
-        width: 50%;
         transform: translateX(-20%);
+        justify-content: flex-end;
```

Comment:
Now might be a good time to remove this dodgy `transform: translateX(-20%)` so the social icons match the main site in being aligned with the right end of the main content grid

Reply:
done